namespaces/referencing-namespace.md

```diff
@@ -0,0 +1,194 @@
+# Referencing Namespace
+
+This namespace defines elements and attributes that may be used to provide the ability to reference and classify elements and namespaces.
```

Comment:
`This namespace ... ` vs. `classify elements and namespaces` for extra clarity I think we need to distinguish these two. Or are they actually the same? What about `This document defines elements ...`